File: login_and_signup_web-master/lib/Model/ListDAActType.dart
class ListDAActType {
  int cqmtransTypeID;
  String cqmtransTypeDesc;
  int zQuoteItemID;
  int nUnitID;
  String unitsDesc;
  int zVendTypeID;

  ListDAActType(
      {this.cqmtransTypeID,
      this.cqmtransTypeDesc,
      this.zQuoteItemID,
      this.nUnitID,
      this.unitsDesc,
      this.zVendTypeID});

  factory ListDAActType.fromJson(Map<String, dynamic> json) {
    return ListDAActType(
        cqmtransTypeID: json['cqmtransTypeId'],
        cqmtransTypeDesc: json['cqmtransTypeDesc'],
        zQuoteItemID: json['zQuoteItemId'],
        nUnitID: json['nUnitId'],
        unitsDesc: json['unitsDesc'],
        zVendTypeID: json['zVendTypeId']);
  }
}

File: AniDAAPI/AniDAAPI/Models/ListDAActType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace AniDAAPI.Models
{
    public partial class ListDAActType
    {
        
        public int CqmtransTypeId { get; set; }
        public string CqmtransTypeDesc { get; set; }
        public int ZQuoteItemId { get; set; }
        public int NUnitId { get; set; }
        public string UnitsDesc { get; set; }
        public int ZVendTypeId { get; set; }

        public ListDAActType()
        {
            ListBillActTrans = new HashSet<ListBillActTrans>();
        }

        public virtual ICollection<ListBillActTrans> ListBillActTrans { get; set; }
    }
}
